utils/config.go

```diff
@@ -0,0 +1,234 @@
+package utils
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/linkerd/linkerd2/testutil"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	defaultNs            = "l5d-conformance"
+	defaultClusterDomain = "cluster.local"
+	defaultVersion       = "stable-2.8.0"
```

Comment:
We're actually doing that only - the latest version of Linkerd is being fetched from the `version.json` endpoint if you have a look at the `parse()` method; however it's possible for that to return an error for any reason, and hence it would just use this statically set `defaultVersion` :)
It still makes sense to remove this variable because in that case, user should just manually set a version

Reply:
Yeah. I don't see why fetch version.json could fail as we heavily depend on it in a lot of other places. We can just remove this and propagate a error! if both are not available i.e version.json and users override.